src/components/HistoryTable/HistoryTableRow/FoldedCommit.tsx

```diff
@@ -14,6 +14,14 @@ import {
 } from "../constants";
 import { RowContainer } from "./styles";
 
+export interface FoldedCommitAnalytics {
+  toggle: "open" | "close";
+}
+
+export const foldedCommitAnalytics = (
+  isVisible: boolean
+): FoldedCommitAnalytics => ({ toggle: isVisible ? "open" : "close" });
```

Comment:
This pattern is a bit confusing. I think it can be avoided by just containing the analytics logic in this file. The url helps differentiate where the event is originating.  (See above comments)